deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -79,10 +79,11 @@ spec:
   graphing:
     enabled: false
     grafana:
-      ingressEnabled: false
-      adminPassword: secret
-      adminUser: root
+      ingressEnabled: true
       disableSignoutMenu: false
+      baseImage: registry.redhat.io/rhel8/grafana:7
```

Comment:
Ah shoot, I think we wanted to change this to `grafana:9`. Might need to get a PR up for that quickly. I have one other thing to review from a docs PR I have open that Chris commented on too.

Reply:
Addressed by https://github.com/infrawatch/service-telemetry-operator/pull/574/commits/28183df3ad5200351d495b7f823a6d0d15dd0897